### PR TITLE
Sample added for Azure in Presentation

### PR DESCRIPTION
**Pull request by Venkateshmuruganandam**
Hi All,
Sample added for following platform:

1. Azure Functions v1 and v4  PPTX to PDF 
2. Azure Functions v1 and v4  PPTX to Image
3. Azure App service Windows and Linux PPTX to PDF
4. Azure App service Windows and Linux PPTX to Image
5. Folder structure change for Find and replace using regex
6. Folder structure change for Find and highlight using regex

Regards
Venkatesh

### CODE REVIEW

File: PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/Azure/Azure_App_Service/Convert-PowerPoint-Presentation-to-Image/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+    {
+        <div class="Common">
+            <div class="tablediv">
+                <div class="rowdiv">
+                    This sample illustrates how to convert PowerPoint document to image using PowerPoint library.
+                </div>
+                &nbsp;
+                <div class="rowdiv" style="border-width: 0.5px;border-style:solid; border-color: lightgray; padding: 1px 5px 7px 5px">
+                    Click the button to view the resultant image document being converted from PowerPoint document.
```
Comment (SuriyaBalamurugan) [inline PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/Azure/Azure_App_Service/Convert-PowerPoint-Presentation-to-Image/Views/Home/Index.cshtml:12]: image file not image document

File: PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/Azure/Azure_Functions/Console_Application/PowerPoint-Presentation-to-Image/Program.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace ConvertPresentationtoImage
```
Comment (SuriyaBalamurugan) [inline PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/Azure/Azure_Functions/Console_Application/PowerPoint-Presentation-to-Image/Program.cs:11]: Use proper sample naming everywhere in this sample

File: PPTX-to-PDF-conversion/Convert-PowerPoint-presentation-to-PDF/Azure/Azure_Functions/Console_Application/PowerPoint-Presentation-to-PDF/Program.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+
+
+namespace ConvertPresentationtoPDF
```
Comment (SuriyaBalamurugan) [inline PPTX-to-PDF-conversion/Convert-PowerPoint-presentation-to-PDF/Azure/Azure_Functions/Console_Application/PowerPoint-Presentation-to-PDF/Program.cs:11]: Use sample name everywhere properly

**Review by SuriyaBalamurugan — CHANGES_REQUESTED** [review]

**Comment by SuriyaBalamurugan** [comment]
@Venkateshmuruganandam - Please give proper title for PR.

File: PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/Azure/Azure_Functions/Console_Application/PowerPoint-Presentation-to-Image/Program.cs (hunk 2 above)
Comment (Venkateshmuruganandam) [reply PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/Azure/Azure_Functions/Console_Application/PowerPoint-Presentation-to-Image/Program.cs]: Updated

File: PPTX-to-PDF-conversion/Convert-PowerPoint-presentation-to-PDF/Azure/Azure_Functions/Console_Application/PowerPoint-Presentation-to-PDF/Program.cs (hunk 3 above)
Comment (Venkateshmuruganandam) [reply PPTX-to-PDF-conversion/Convert-PowerPoint-presentation-to-PDF/Azure/Azure_Functions/Console_Application/PowerPoint-Presentation-to-PDF/Program.cs]: Updated

File: PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/Azure/Azure_App_Service/Convert-PowerPoint-Presentation-to-Image/Views/Home/Index.cshtml (hunk 1 above)
Comment (Venkateshmuruganandam) [reply PPTX-to-Image-conversion/Convert-PowerPoint-presentation-to-Image/Azure/Azure_App_Service/Convert-PowerPoint-Presentation-to-Image/Views/Home/Index.cshtml]: Updated

**Review by Suriya-Balamurugan — APPROVED** [review]
I don't have any concerns in this code changes.

**Review by MohanaselvamJothi — APPROVED** [review]

**Pull request merged**